react/features/toolbox/components/web/Toolbox.js

```diff
@@ -1007,9 +1007,9 @@ class Toolbox extends Component<Props, State> {
                 && <SharedDocumentButton
                     key = 'etherpad'
                     showLabel = { true } />,
-            this._shouldShowButton('videobackgroundblur')
-                && <VideoBlurButton
-                    key = 'videobackgroundblur'
+            this._shouldShowButton('select-background')
```

Comment:
Add some backwards compatibility check here. It should still work with the old button name.